manifests/prow/services/ghproxy.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ghproxy
+  name: ghproxy
+  namespace: prow
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8888
+  - name: metrics
```

Comment:
It seems there is a metrics port, but there is no ServiceMonitor or PodMonitor. Do you plan to add ServiceMonitor or PodMonitor?

Reply:
Same as with the `health` port, the metrics port is documented on all the commands but not implemented. In some cases, the port is not even listening.

Reply:
OMG xD

Reply:
Let's wait for further improvements on upstream!